index.html

```diff
@@ -86,28 +88,33 @@ <h5><a href="http://www.marvel.com">Marvel</a></h5>
       </li>
       <li ng-repeat="comic in main.comics track by comic.id" class="comic-container" ng-class="{expanded: comic.id === main.expandedComicId}" ng-style="comic.containerStyles">
         <div class="scroll-anchor"></div>
+        <div class="popover"><div>#{{::comic.issue}}</div></div>
         <div class="comic z-depth-5 hoverable" ng-style="comic.styles" ng-class="{stickyTop: comic.classes.stickyTop, stickyRight: comic.classes.stickyRight, stickyBottom: comic.classes.stickyBottom, stickyLeft: comic.classes.stickyLeft}">
           <div class="panel-left">
-            <a title="#{{::comic.issue}}" ng-click="main.toggleExpandComic(comic)" class="waves-effect cover-thumbnail" data-toggle="tooltip">
+            <a ng-click="main.toggleExpandComic(comic)" class="waves-effect cover-thumbnail">
               <img class="responsive-img" ng-src="http://www.spirton.com/images/comics/{{::comic.image}}.jpg"></img>
             </a>
             <ul class="pagination">
-              <li class="waves-effect" ng-class="{disabled: !main.prevCollectionFirstComic}" data-toggle="tooltip" title="{{main.prevCollection.title}}">
+              <li class="waves-effect" ng-class="{disabled: !main.prevCollectionFirstComic}">
+                <div class="popover"><div>{{main.prevCollection.title}}</div></div>
```

Comment:
This has the effect that if the comic is the first one in the first collection, the chevron and arrows are greyed out but are still active and a `-` and `#` popover contents are shown.  Should no popover display if there are no earlier collections, or comics, or later collections, or comics?

Reply:
Yup will fix